moveit.rosinstall

```diff
@@ -1,55 +1,49 @@
 # This file is intended for users who want to build MoveIt! from source.
 # Used with wstool, users can download source of all packages of MoveIt!.
 
+- git:
+    local-name: moveit_msgs
+    uri: https://github.com/ros-planning/moveit_msgs
+    version: ros2
 - git:
     local-name: moveit2
-    uri: https://github.com/ros-planning/moveit2.git
+    uri: https://github.com/ros-planning/moveit2
     version: master
 - git:
-    local-name: moveit_msgs
-    # TODO(mlautman): update to ros-planning once the package has been merged back in
-    uri: https://github.com/AcutronicRobotics/moveit_msgs
-    version: ros2
+    local-name: joint_state_publisher
+    uri: https://github.com/ros/joint_state_publisher
+    version: ros2-devel
+  # TODO(henningkayser): Change to 'ros-planning/ros2' once https://github.com/ros-planning/random_numbers/pull/18 is merged
+- git:
+    local-name: random_numbers
+    uri: https://github.com/PickNikRobotics/random_numbers
+    version: ros2-devel
+  # TODO(henningkayser): merge dependencies below into upstream pacages
 - git:
     local-name: geometric_shapes
-    # TODO(mlautman): update to ros-planning once the package has been ported
```

Comment:
This TODO should not be removed. We should be merging the PRs in ros-planning, why are we instead switching to PickNikRobotics? We have full control over the ros-planning repo, lets focus on that